sos/plugins/__init__.py

```diff
@@ -1026,6 +1026,17 @@ def report(self):
         else:
             return html
 
+    def check_process_by_name(self, pr):
```

Comment:
This adds an entire new interface to the `Plugin` class: it needs to be in a separate commit with an adequate explanation in the commit message.

Reply:
The interface itself is fine, although I would suggest using something like `process`, `name`, or `command` instead of `pr` - there's only a single method argument here after self, so we're not getting short on space and API argument names should be as descriptive as possible where space allows.

Reply:
submitted https://github.com/sosreport/sos/pull/1186 for the new method